mopidy_spotify/library.py

```diff
@@ -63,20 +63,13 @@ def _lookup_album(self, sp_link):
     def _lookup_artist(self, sp_link):
         sp_artist = sp_link.as_artist()
         sp_artist_browser = sp_artist.browse(
-            type=spotify.ArtistBrowserType.NO_TRACKS)
+            type=spotify.ArtistBrowserType.FULL)
```

Comment:
The libspotify docs say
> There is currently no built-in functionality available for getting the albums belonging to an artist. For now, just iterate over all tracks and note the album to build a list of all albums

Isn't using `FULL` the only way to get all the tracks? I know the `tophit` tracks are supposed to be missing but if there really are other results missing then that's hopeless. Probably worth looking into and verifying.  Although arguably you're potentially loading a lot of useless track data (various artists, compilations) that will just be filtered out. This kind of lookup is far superior in the web API; it's a shame they hate libspotify so much.